.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,32 @@
-### Issue being fixed or implemented  
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- Pull request titles must use the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) format -->
 
-Please include a summary of what is the issue being fixed, along with the relevant motivation and context.
-List any dependencies that are required for this change.
+## Issue being fixed or feature implemented
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
 
-### What was done  
 
-Please include a summary of the change
+## What was done?
+<!--- Describe your changes in detail -->
 
-### How Has This Been Tested?
 
-Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
 
-- [ ] Test A
-- [ ] Test B
 
-### Notes  
+## Breaking Changes
+<!--- Please describe any breaking changes your code introduces and verify that -->
+<!--- the title includes "!" following the conventional commit type (e.g. "feat!: ..."-->
 
 
-### Checklist:
-
-- [ ] My code follows the style guidelines of this project
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
 - [ ] I have performed a self-review of my own code
 - [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have added or updated relevant unit/integration/functional/e2e tests
 - [ ] I have made corresponding changes to the documentation
+
+**For repository code-owners and collaborators only**
+- [ ] I have assigned this pull request to a milestone
```

Comment:
Is there any plan to use milestones here like the other platform-related repos?

Reply:
Not that I'm aware of, but we did discuss about moving into that so we could highlight some wanted changes to the community so they are aware, so I guess we would go this route in the future. 
Right now, we do not.

Reply:
Okay, I will remove for now and we can add it back later if we start using milestones